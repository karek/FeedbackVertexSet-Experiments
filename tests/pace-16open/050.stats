49,84,13
