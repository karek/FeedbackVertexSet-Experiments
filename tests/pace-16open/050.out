7
9 2 1 19 40 30 38 
